lingui.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('@lingui/conf').LinguiConfig} */
+module.exports = {
+	locales: ["en", "fr"],
+	sourceLocale: "en",
+	catalogs: [{
+	   path: "src/locales/{locale}/messages",
+	   include: ["src"]
+	}],
+	format: "po"
```

Comment:
The format is by default po, which is used by the GNU gettext. They also have json and csv format options included, but custom formatters can also be used.